pages/osx/pod.md

```diff
@@ -0,0 +1,31 @@
+# pod
+
+> Dependency manager for Swift and Objective-C Cocoa projects.
+
+- Create a Podfile for the current project with the default contents:
+
+`pod init`
+
+- Download all pod defined in Podfile and create an Xcode Pods library project in `./Pods`:
+
+`pod install`
+
+- Download a new pod for the current project and add it to the Podfile:
+
+`pod install {{pod_name}}`
+
+- Show the outdated pods in the current `Podfile.lock`:
```

Comment:
Good point, but I will change it to "currently installed" instead, because pods can be defined in the Podfile but not actually installed (i.e. present in the `Podfile.lock`) until `pod install` is run.